cpp/test/data/super_capacitor.info

```diff
@@ -2,7 +2,15 @@ type SuperCapacitor
 dim  2
 
 geometry {
-    #include "read_mesh.info"
+    type supercapacitor
+    
+    anode_collector_thickness    5.0e-4 ; [centimeter]
+    anode_electrode_thickness   50.0e-4 ; [centimeter]
+    separator_thickness         25.0e-4 ; [centimeter]
+    cathode_electrode_thickness 50.0e-4 ; [centimeter]
+    cathode_collector_thickness  5.0e-4 ; [centimeter]
+    geometric_area              25.0e-2 ; [square centimeter]
+    tab_height                   5.0e-4 ; [centimeter]
 }
 
 solid_potential_component  0
```

Comment:
Only solid_potential_component and liquid_potential_component are used but personally I don't think this is a good idea for them to be in the input file. I think this the kind of things that should be centralized in a file. Changing the order of the component won't affect the properties of the matrix so there is no point in having these as input parameters.